src/helpers/ast/is-literal.js

```diff
@@ -0,0 +1,4 @@
+// Literals, and `undefined`, are literals
+export default function isLiteral(t, node) {
+  return t.isLiteral(node) || (t.isIdentifier(node) && node.name === "undefined");
+}
```

Comment:
It's fine to create a helper like this, but I don't like calling `undefined` a literal like this.